data/scrape/lib/codes.py

```diff
@@ -174,7 +174,14 @@ def main() -> None:
             wiktionary_code = _scrape_wiktionary_language_code(
                 wiktionary_name.replace(" ", "_")
             )
-            iso639_lang = iso639.Language.match(wiktionary_code)
+
+            try:
+                iso639_lang = iso639.Language.match(wiktionary_code)
+            except iso639.language.LanguageNotFoundError:
+                logging.warning(
+                    "Could not find language with code %s", wiktionary_code
+                )
+
```

Comment:
This is admittedly a pet peeve of mine but I absolutely loathe blank lines within functions; they make it hard for me to see where one functions begins and another ends. Can you remove the new ones on lines 177 and 184? If you want to set off the code here from the rest of the function, a comment would be fine.

Reply:
Sure, np